lib/stdlib/doc/src/maps.xml

```diff
@@ -142,6 +142,20 @@
       </desc>
     </func>
 
+    <func>
+      <name name="foreach" arity="2" since=""/>
```

Comment:
```suggestion
      <name name="foreach" arity="2" since="OTP @OTP-17179@"/>
```

Please include this ticket number so that the since tag will be correct